tests/test_ghost.py

```diff
@@ -45,7 +45,7 @@ def build_contract(self, preprocess=False):
             self.CONTRACT = hash160(output)
 
     def deploy_contract(self, engine):
-        cc = TestContract(self.CONTRACT_PATH_NEF)
+        cc = TestContract(self.CONTRACT_PATH_NEF, self.CONTRACT_PATH_JSON)
```

Comment:
this was not changed on test_gm.py, any reason?

Reply:
I do not see this method signature in test_gm.py